react/src/components/promo/SprkPromo.stories.js

```diff
@@ -37,19 +69,37 @@ defaultStory.story = {
 
 export const flag = () => (
   <SprkPromo
-    additionalClasses="sprk-o-Stack--split@s"
-    imgSrc="https://spark-assets.netlify.app/spark-logo-updated.svg"
-    imgAlt="Spark Design System Logo"
-    imgLinkHref="#nogo"
-    imgLinkAnalytics="promo-3-img-link-analytics"
-    imgLinkIdString="promo-3-img-link"
-    idString="promo-3"
-    isFlag
+    idString="flag-promo"
     hasBorder
+    isFlag
+    additionalClasses="sprk-o-Stack--split@s"
   >
-    Lorem ipsum dolor. Sit amet pede. Tempus donec et. Suspendisse id inventore
-    integer eum non enim diam habitant. Maecenas nunc per lacus neque egestas.
-    Diam quod curabitur.
+    <SprkStackItem additionalClasses="sprk-o-Stack__item--fourth@s">
+      <SprkLink variant="unstyled" href="#nogo">
+        <img
+          className="sprk-c-Promo__image sprk-c-Promo__image--flag"
+          alt="Learn more about Spark Design System."
+          src="https://spark-assets.netlify.app/spark-logo-updated.svg"
+        />
+      </SprkLink>
+    </SprkStackItem>
+
+    <SprkStackItem
+      additionalClasses="
+        sprk-c-Promo__content
+        sprk-o-Stack__item--three-fourths@s
+      "
+    >
```

Comment:
Out of curiosity, is there a reason Angular adds the promo classes using directives (`sprkPromoContent` etc) while React just adds the classes directly? 

Why not this:
```
<SprkPromoContent
   additionalClasses="sprk-o-Stack__item sprk-o-Stack__item--three-fourths@s"
>
```

or this:

```
<div
   additionalClasses="sprk-c-Promo__content sprk-o-Stack__item sprk-o-Stack__item--three-fourths@s"
>
```

Reply:
I'm glad you asked that. It's a difference in the functionality of react/angular. Angular can have directives on a component so we can a better (in my option) setup. With React, we have to choose one of the three options, the way I have it, or the two you mentioned. I chose the way I have it because using the Stack layout objects to build the pattern felt the most compositional and efficient. Stack has a ton of inputs built in like splitAt, itemSpacing, etc. These inputs are easier to use than having to remember the very verbose Stack classes. Especially the --split@xxs. So Stack is more complicated than just the simple one class from Promo content. So we reap benefits from using the Stack component and adding the one promo class. Thanks again for noticing this and asking it. Looking forward to your thoughts! 🎉😃 @bobwise

Reply:
I agree with your decision. `<SprkStack>` is a million times more reusable than `<SprkPromoContent>` and just making it a `div` with `additionalClasses` defeats the purpose of using React.

Reply:
Awesome! Thank you! I am super glad that you noticed that and brought it up. It shows that you really thought about this work and I appreciate it. @bobwise